### PR TITLE
Issue 1521: Move the images for the Nuget (Visual Studio) example from the old wiki to the new one

### LINKED ISSUE #1521: Move the images for the Nuget (Visual Studio) example from the old wiki to the new one
# Task Description

We have migrated our wiki from [the strongbox project](https://github.com/strongbox/strongbox/wiki) to [strongbox-docs](https://strongbox.github.io/) project. We appear to have some images still pointing to the old wiki and this prevents us from being able to shut down the old wiki, which is now obsolete and might be misleading people.

# Tasks

The following tasks will need to be carried out:
* [ ] Clone the docs from `git@github.com:strongbox/strongbox-docs` 
* [ ] Then move the images necessary images from `wiki/**` under `resources/images/nuget/` to `docs/assets/images` under the [`strongbox-docs`](https://github.com/strongbox/strongbox-docs/) repository. Make sure to preserve git history!
* [ ] Change the link for the images in the [hello-strongbox-nuget-visual-studio](https://github.com/strongbox/strongbox-examples/tree/master/hello-strongbox-nuget-visual-studio) to point to the new locations.
* [ ] Check if there are other images which have not been migrated and report/fix them.

# Useful Links

* [strongbox-docs](https://github.com/strongbox/strongbox-docs/)

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @carlspring
  * @fuss86
  * @sbespalov
  * @steve-todorov

### PR DESCRIPTION
**Pull request by JekRock**
Fixes strongbox/strongbox#1521

Issue 1521: Move the images for the Nuget (Visual Studio) example from the old wiki to the new one

- Change link for the images to point to the new locations

### CODE REVIEW

File: hello-strongbox-nuget-visual-studio/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To manage packages, you'll need to configure NuGet to acccess your storages by p
 3. To add a source, select +, edit the name, enter the URL or path in the Source control, and select Update. The source now appears in the selector drop-down.
 4. The `nuget-public` group repository should be at the top of the list.
 
-![NuGet Package Resources](https://github.com/strongbox/strongbox/wiki/resources%2Fimages%2Fnuget%2FNuGet-Package-Resources.png "Add new NuGet Package Resources")
+![NuGet Package Resources](https://github.com/strongbox/strongbox/docs/assets/images/NuGet-Package-Resources.png "Add new NuGet Package Resources")
```
Comment (carlspring) [inline hello-strongbox-nuget-visual-studio/README.md]: I'm afraid these links are incorrect. They should be pointing to the published ones under https://strongbox.github.io/. Could you please correct these? Thanks! :)

Comment (JekRock) [reply hello-strongbox-nuget-visual-studio/README.md]: @carlspring so it should be like https://strongbox.github.io/assets/images/*.png ?

Comment (steve-todorov) [reply hello-strongbox-nuget-visual-studio/README.md]: That's correct:
[`./docs/assets/**`](https://github.com/strongbox/strongbox-docs/tree/master/docs/assets) == `https://strongbox.github.io/assets/**`

**Review by steve-todorov — APPROVED** [review]
LGTM! :)

**Comment by carlspring** [comment]
Hi @JekRock ,

Thanks for your help with this! Congratulations on your first pull request! :)

**Comment by JekRock** [comment]
@carlspring Thanks!

**Pull request merged**